src/App.tsx

```diff
@@ -1,109 +1,21 @@
 import React from 'react';
 
 import './App.scss';
+import { PostList } from './components/PostList';
 
-// import postsFromServer from './api/posts';
-// import commentsFromServer from './api/comments';
-// import usersFromServer from './api/users';
+import postsFromServer from './api/posts';
+import commentsFromServer from './api/comments';
+import usersFromServer from './api/users';
 
 export const App: React.FC = () => (
   <section className="App">
     <h1 className="App__title">Static list of posts</h1>
 
-    <div className="PostList">
-      <div className="PostInfo">
-        <div className="PostInfo__header">
-          <h3 className="PostInfo__title">qui est esse</h3>
+    <PostList
+      posts={postsFromServer}
+      users={usersFromServer}
+      comments={commentsFromServer}
```

Comment:
PostList should accept only posts 
![image](https://user-images.githubusercontent.com/97547824/228636108-def9b3de-1ef0-4d49-a839-d3215e93071a.png)
Do it before passing it to the PostList component